### DISCUSSION TITLE
Do the same with Azure Data Studio

### DISCUSSION THREAD

**Discussion by blavoie** [coder/code-server#2385] [Ideas]
Hi!

Very cool project and works well, but wondering if it's possible to do the same with Azure Data Studio?

It seems to use the same underlying stack, would be nice to launch it as server too.

Thanks
Bruno

**Comment by blavoie** [comment]
Would this be a complex task?

**Reply by code-asher** [reply]
From what I can tell all we'd have to do is apply the patch we already have although I haven't tested this.

It'd double the testing we currently do manually though; might make sense to wait until we have our end to end tests in place.

**Reply by gfody** [reply]
I'm interested in trying to do this. is the idea to fork https://github.com/microsoft/azuredatastudio and apply the patches from https://github.com/coder/code-server/tree/main/patches?

**Reply by code-asher** [reply]
That seems like the right strategy to me.
